backend/pages/models.py

```
@@ -6,6 +6,9 @@
 from .utils.data_processing import run_pipeline_for_sheet, extract_section_name
 from .utils.chart_classification import ADDITIONAL_PROCESSING_PIPELINE
 from django.contrib.auth import get_user_model
+from django.db import models
```

Ruff (0.11.9)

Redefinition of unused `models` from line 4

Remove definition: `models`

(F811)